src/Microsoft.SqlTools.ServiceLayer/ObjectManagement/ObjectManagementService.cs

```diff
@@ -87,12 +85,11 @@ internal async Task HandleRenameRequest(RenameRequestParams requestParams, Reque
         /// Method to get the sql object, which should be renamed
         /// </summary>
         /// <param name="requestParams">parameters which are required for the rename operation</param>
-        /// <param name="connection">the sqlconnection on the server to search for the sqlobject</param>
+        /// <param name="connection">the server connection on the server to search for the sqlobject</param>
         /// <returns>the sql object if implements the interface IRenamable, so they can be renamed</returns>
-        private IRenamable GetRenamable(RenameRequestParams requestParams, SqlConnection connection)
+        private IRenamable GetRenamable(RenameRequestParams requestParams, ServerConnection connection)
         {
-            ServerConnection serverConnection = new ServerConnection(connection);
```

Comment:
@Charles-Gagnon, here, we are not including the access token information when constructing the server connection.